Breeding-Insight/bi-docker-stack
# [BI-2101] added SMTP auth variables to compose file

## Description

Environment variables are passed into individual containers by docker compose, the `EMAIL_RELAY_LOGIN` and `EMAIL_RELAY_PASSWORD` are needed for bi-api to send transactional email.

I tested without these variables in my .env which allowed to continue using mailhog, and I tested with these variables in my .env which allowed me to send real transactional email.

## Testing

The changes made in https://github.com/Breeding-Insight/bi-docker-stack/pull/45 are required, I recommend merging that branch into your local branch for testing.
1. `git checkout bug/BI-2118`
2. `git checkout feature/BI-2101-fix`
3. `git merge bug/BI-2118`

To test, you'll need to 
1.  set `EMAIL_RELAY_LOGIN` and `EMAIL_RELAY_PASSWORD` in your .env file (use shared credentials)
2. run bi-docker-stack locally (tip: use the `--build` flag with docker-compose so it doesn't use a cached biproxy image)
3. test creating a new user, ensure email is delivered
